src/processor/operator/persistent/reader/parquet/interval_column_reader.cpp

```diff
@@ -0,0 +1,34 @@
+#include "processor/operator/persistent/reader/parquet/interval_column_reader.h"
+
+namespace kuzu {
+namespace processor {
+
+common::interval_t IntervalValueConversion::readParquetInterval(const char* input) {
```

Comment:
Ditto. I'm curious why codecov reports uncovered lines here. I thought this function should be used during reading.

Reply:
I think this is a false positive.